src/server/debug.rs

```diff
@@ -755,9 +755,11 @@ impl MvccChecker {
     pub fn check_mvcc(&mut self, wb: &WriteBatch) -> Result<()> {
         loop {
             // Find min key in the 3 CFs.
-            let mut key = MvccChecker::min_key(None, &self.default_iter, truncate_ts);
+            let mut key = MvccChecker::min_key(None, &self.default_iter, |k| {
+                Key::truncate_ts_for(k).unwrap()
+            });
             key = MvccChecker::min_key(key, &self.lock_iter, |k| k);
-            key = MvccChecker::min_key(key, &self.write_iter, truncate_ts);
+            key = MvccChecker::min_key(key, &self.write_iter, |k| Key::truncate_ts_for(k).unwrap());
```

Comment:
just unwrap() is that okay?

Reply:
Yes, when there is indeed codec errors, it will behave the same as before. Previous a panic will be raised in the function and now the function returns a Result so we unwrap here.

Reply:
so panic is acceptable here?

Reply:
There might be space for improvement. I mean, previously it would panic, now it will panic, nothing is changed.